exams/pearson_test.py

```diff
@@ -57,31 +80,55 @@ def test_profile_country_to_alpha3_invalid_country(self):  # pylint: disable=no-
             profile = ProfileFactory.create()
         profile.country = 'XXXX'
         with self.assertRaises(InvalidProfileDataException):
-            _profile_country_to_alpha3(profile)
+            CDDWriter.profile_country_to_alpha3(profile)
+
 
-    def test_write_profiles_cdd_no_profiles(self):  # pylint: disable=no-self-use
+class ExamPearsonProfileTest(TestCase):
```

Comment:
I think you need to reorganize all the tests here around the new classes you wrote and make tests first for the base class and its methods and then test that the subclasses are instantiated in the right way.